synapse/lib/cell.py

```diff
@@ -60,6 +60,8 @@
 
 logger = logging.getLogger(__name__)
 
+NEXUS_VERSION = 1
```

Comment:
Do we want this to be a major, minor tuple?

Reply:
Like using the synapse major and minor version where the nexus handlers last changed?

Reply:
Updated this to be a tuple